wire/net/test/connhub_internal_test.go

```diff
@@ -51,7 +51,7 @@ func TestConnHub_Create(t *testing.T) {
 			ct.Stage("accept", func(rt pkgtest.ConcT) {
 				conn, err := l.Accept(ser)
 				assert.NoError(err)
-				require.NotNil(rt, conn)
+				assert.NotNil(rt, conn)
```

Comment:
As mentioned above: Don't replace `require` with `assert` (which returns a bool instead of panicking).